### DISCUSSION TITLE
(proposal) Add _unchecked prop for Switch and Checkbox

### DISCUSSION THREAD

**Discussion by fzn0x** [chakra-ui/chakra-ui#8435] [Ideas] [answered]
### Description

Add _unchecked prop

### Problem Statement/Justification

You can only change unchecked checkbox or switch styles by modifying the default style or use `sx`

```jsx
<Switch colorScheme='blue' sx={{ 'span.chakra-switch__track:not([data-checked])': { bg: 'blue.200' } }} />
```

### Proposed Solution or API

```jsx
<Switch colorScheme='blue' _unchecked={{ bg: 'blue.200'  }} />
```

### Alternatives

N/A

### Additional Information

We can treat _unchecked as 'span.chakra-switch__track:not([data-checked])' (for switch) for example.

**Comment by segunadebayo** [comment, marked as answer]
There is no `_unchecked` state. Unchecked is the default look of the control.

Put the off styles in `base`. Put the on styles in `_checked` on `Switch.Control` or `Checkbox.Control`.
